packages/react-native/src/primitives/Checkbox/types.ts

```diff
@@ -19,3 +19,5 @@ export interface CheckboxProps<T>
 export interface CheckboxStyle {
   container: ViewStyle;
 }
+
+export type LabelPosition = 'start' | 'end' | 'top' | 'bottom';
```

Comment:
Since `Radio` will need the `LabelPosition` type as well, maybe we can put it in a shared types file?

Reply:
Done!